docs/source/changelog.rst

```diff
@@ -2,6 +2,18 @@
 
 Changelog
 ---------
+**Future Release**
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+        * Miscellaneous changes (:pr:`641`)
+    * Testing Changes
+        * Miscellaneous changes ()
```

Comment:
No PR number?

Reply:
I left the Miscellaneous there since normal testing changes should recorded there

If we don't have any this dev cycle I'll remove it in the release PR